README.md

```diff
@@ -1 +1,168 @@
-# ruby_catalog
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b>[Catalog of my things] is a project at Microverse. It's an app that keeps records of books, games, and music album.
+  
+  Video Presntation link(https://drive.google.com/file/d/1tBGtpw2K507swHiRJKkgwyR2mdt-b4Kj/view?usp=drive_link)</b></h3>
+
```

Comment:
- [ ] Please I couldn't view your presentation because I have no access to view it so I think you should make your presentation video public so that the next reviewer can view your presentation :pray:

Reply:
Done. Thanks